testing/display_list_testing.cc

```diff
@@ -287,6 +287,17 @@ static std::ostream& operator<<(std::ostream& os, const SkTextBlob* blob) {
   return os << "&SkTextBlob(ID: " << blob->uniqueID() << ", " << blob->bounds() << ")";
 }
 
+static std::ostream& operator<<(std::ostream& os,
```

Comment:
Could be a utility in text_frame.cc

Reply:
TextFrame might want to have a more fully featured streamer. This one just needs enough to hold it's place in the debugging output. The Text objects and Path just dump out some useful info here like bounds.